engines/basic_resources/app/controllers/basic_resources/api/v1/memberships_controller.rb

```diff
@@ -61,8 +60,12 @@ def destroy
 
         private
 
+        def group
+          Group.find(params[:group_id]) if params.key?(:group_id)
```

Comment:
I'm wondering if the filter parameter should be `basic_resource_group_id` instead of `group_id`.

I first thought that maybe we could hide behind the endpoint the concept of `basic_resource_*` and make the clients agnostic about that. But then, working from the client side, I saw that at the end when you receive `Membership` resources you need to deal with the `basic_resource_*` from the schema. You just can't avoid to *leak* that concept to the clients.

What do you think? Rename to `params[:basic_resource_group_id]`?